engines/cruise/detection.cpp

```diff
@@ -196,16 +195,19 @@ static const CRUISEGameDescription gameDescriptions[] = {
 class CruiseMetaEngine : public AdvancedMetaEngine {
 public:
 	CruiseMetaEngine() : AdvancedMetaEngine(Cruise::gameDescriptions, sizeof(Cruise::CRUISEGameDescription), cruiseGames) {
-		_singleId = "cruise";
 		_guiOptions = GUIO2(GUIO_NOSPEECH, GUIO_NOMIDI);
 	}
 
+	const char *getEngineId() const {
+		return "cruise";
+	}
+
 	virtual const char *getName() const {
-		return "CruisE";
+		return "Cinematique evo 2";
 	}
 
 	virtual const char *getOriginalCopyright() const {
-		return "Cruise for a Corpse (C) Delphine Software";
+		return "Cinematique evo 2 (C) Delphine Software";
```

Comment:
Interesting. Above you were changing Chewy to the game name, and here you're doing the opposite. Is there some reasoning behind this inconsistency?

Reply:
I guess it's to be consistent with the cine engine.